src/main/java/core/basesyntax/dao/FruitTransactionDao.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.dao;
+
+import java.util.Map;
+import java.util.Set;
+
+public interface FruitTransactionDao {
+    void add(String fruit, int quantity);
+
+    void update(String fruit, int quantity);
+
+    int getQuantity(String fruit);
+
+    Set<Map.Entry<String, Integer>> getEntries();
```

Comment:
why do you use entries?

Reply:
to get info from storage and then create a report based on this info

Reply:
not best way, try to use better type what you need from storage